### ISSUE TITLE
Warzone 2 and mw3

### ISSUE THREAD

**Issue by hamadtareen1** [Nukem9/dlssg-to-fsr3#66]
<img width="673" alt="Screenshot 2023-12-19 041158" src="https://github.com/Nukem9/dlssg-to-fsr3/assets/154284691/16a97cd6-4319-4c71-a5aa-bf8ab0fa10bb">

how do i get it working in warzone and mw3? i did all the steps in the readme file but i dont get any message on startup and there is no setting to turn on frame generation.

**Comment by hamadtareen1** [comment]
<img width="490" alt="Screenshot 2023-12-19 044029" src="https://github.com/Nukem9/dlssg-to-fsr3/assets/154284691/4239ecf3-e610-4af8-b66e-f21900310f3d">



got it working but every once in a while it gives this error and crashes. any workaround?

**Comment by Lysergic-Acid** [comment]
The mod says not to use it with multiplayer games, so.

**Comment by hamadtareen1** [comment]
> The mod says not to use it with multiplayer games, so.

Yes but will it be available for multiplayer anytime soon? Can it be fixed? There’s no harm
In using it for multiplayer. It’s only necessary.

**Comment by Pretentiousness** [comment]
> Yes but will it be available for multiplayer anytime soon? Can it be fixed? There’s no harm in using it for multiplayer. It’s only necessary.
the way mod injects fsr 3 in place of dlss3 can get detected by anti-cheat software and get you banned, hence not recommended to use it. if you still wish to use it then the author will not be responsible for any bans or actions taken against you.
